sdk/documentintelligence/ai-document-intelligence-rest/package.json

```diff
@@ -20,17 +21,14 @@
       "browser",
       "react-native"
     ],
-    "selfLink": false,
-    "project": "./tsconfig.src.json"
+    "selfLink": false
   },
   "type": "module",
   "keywords": [
     "node",
     "azure",
     "cloud",
-    "typescript",
-    "browser",
-    "isomorphic"
```

Comment:
nitpick: why are these being removed?

Reply:
service doesn't care about browsers, we did not invest time in testing there.